ocs_ci/templates/DR/secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+data:
+  key: ""
+metadata:
+  name: PLACEHOLDER
+  namespace: PLACEHOLDER
+  annotations:
+    appnames: "busybox_app1-1,busybox_app1-2,busybox_app1-3,busybox_app1-4,busybox_app1-5,busybox_app1-6,busybox_app1
+-7,busybox_app1-8,busybox_app1-9,busybox_app1-10"
+type: Opaque
```

Comment:
Why is this secret required ?

Reply:
That's a required step when we deploy discovered apps with recipe, am i missing any context here ?

Reply:
can you check i never heard of this requirement